distributed/tests/test_client_executor.py

```diff
@@ -213,18 +213,13 @@ def test_unsupported_arguments(client, s, a, b):
 def test_retries(client):
     args = [ZeroDivisionError("one"), ZeroDivisionError("two"), 42]
 
-    with client.get_executor(retries=5, pure=False) as e:
+    with client.get_executor(retries=6, pure=False) as e:
         future = e.submit(varying(args))
         assert future.result() == 42
 
-    with client.get_executor(retries=4) as e:
+    with client.get_executor(retries=1) as e:
         future = e.submit(varying(args))
-        result = future.result()
-        assert result == 42
-
-    with client.get_executor(retries=2) as e:
-        future = e.submit(varying(args))
-        with pytest.raises(ZeroDivisionError, match="two"):
```

Comment:
Any reason to change these retries?

Reply:
This test is extremely sensitive to how workers are selected on an idle cluster. It uses this stateful `varying` utility, which changes its behavior depending on how many times it's been called on that worker.

It had to be changed when the idle-round-robin behavior was added: https://github.com/dask/distributed/pull/4638/files#diff-59af67191283f0c64a3be8ce1f344f49b9d025f8264b77fba5c8250865bde433

So I've had to change it again since it's being removed.